src/python/espressomd/reaction_ensemble.pyx

```diff
@@ -11,34 +11,32 @@ class WangLandauHasConverged(Exception):
 
 cdef class ReactionAlgorithm(object):
     """
-    This class provides the Reaction Ensemble algorithm, the Wang-Landau
-    Reaction Ensemble algorithm and the constant pH method. Initialize the
-    reaction ensemble by setting the standard pressure, temperature, and the
-    exclusion radius.
+    This class provides the following algorithms: ReactionEnsemble, ConstantpHEnsemble, and
+    WangLandauReactionEnsemble.
+    Initialize the reaction ensemble with the given algorithm by setting the
+    standard pressure, temperature, and the exclusion radius. 
 
     Parameters
     ----------
     standard_pressure : :obj:`float`
-                        The pressure in simulation units where the reactions
-                        should occur. This is an input parameter of the
-                        reaction ensemble.
+                        The standard pressure (in simulation units)
+                        at which the reaction constant was determined.
+                        Note that if you provide the reaction constant :math:`K_c`
+                        as described in the user guide, this parameter should be kept at its default value.
+                        This paramater is retained for backward compatibility with earlier
+                        implementation, where the pressure-based constaht :math:`K_p` was
```

Comment:
The user guide only deals with a dimensionless reaction constant K (K_p is not mentioned there). Please also name it this way here.

Reply:
Sorry i overlooked parts. But how is K=Exp(...) related to K_p?

Reply:
I don't see that there is a statement K=K_p although it is used in that way. After all this only holds for ideal gases but all the reaction constants which are provided in the reaction ensemble are ideal Gas reaction constants which are altered by the Interactions. Can we introduce a statement like this or so you see Problems with it?

Reply:
You are right, K_p is not defined anywhere. At the same time, though K_p = prod_i (p_i / p_standard)^{nu_i} is well defined only if the system is a gas, and it is equal to K only if the system is an ideal gas. Regarding the message of this note, saying that standard_pressure is needed only in gas-phase reactions, I would leave it as is. I expect that most users need not bother, and those who need to study gas-phase reactions should be familiar with K_p.